tools/dataset_converters/waymo_converter.py

```diff
@@ -604,29 +400,301 @@ def cart_to_homo(self, mat):
             raise ValueError(mat.shape)
         return ret
 
+    def create_waymo_info_file(self, frame, file_idx, frame_idx, file_infos):
```

Comment:
docstring, please also specify the format of info or if we describe the info in this PR message, add a link to this PR